src/block-management/block-toolbar.js

```diff
@@ -38,29 +38,38 @@ export class BlockToolbar extends Component<PropsType> {
 		} = this.props;
 
 		return (
-			<View style={ styles.container }>
+			<View style={ styles.container } accessible={ false } accessibilityLabel="View Add block">
 				<ScrollView
+					accessible={ false }
+					accessibilityLabel="Scrollview Add block"
 					horizontal={ true }
 					showsHorizontalScrollIndicator={ false }
 					keyboardShouldPersistTaps={ 'always' }
 					alwaysBounceHorizontal={ false }
 					contentContainerStyle={ styles.scrollableContent }
 				>
-					<Toolbar>
+					<Toolbar accessible={ true }
+						accessibilityLabel={ 'Toolbar Add block' }>
 						<ToolbarButton
-							label={ __( 'Add block' ) }
+							title={ __( 'Add block' ) }
 							icon="insert"
+							accessible={ true }
+							accessibilityLabel={ __( 'Add block' ) }
```

Comment:
Since this is a localized value it might give us a hard time if we want to run UI tests in other languages. As far as I see we are using non localized values in UI tests to query this element. In that case we'd need to localize our query word there also. Another option could be just using testID and assigning non localized values to it.

Reply:
I don't know why I localized here really, more than likely just the result of a copy paste. Thanks, I'll remove this. Also, testID isn't supported on Android(doesn't populate the content description attribute) so that's why I've removed all references to it and used the accessibilityLabel instead.

Reply:
Actually I had offered using non localized words only if we could use testID prop but we can't :( In this case we should continue using localized values, but, we should be using localized values while writing UI tests also. 

I wonder somehting, is it possible to set accessible={ false } and reach accessibilityLabel from UI tests at the same time?

I did a quick search on testID, it looks like react native translates testID into View tags on Android side. But Appium doesn't support querying with tags, so Appium and react-native behave incompatible about this :( It looks possible with Espresso btw.

I saw several PRs that are proposing to change this behavior and translating testIDs into `respurce-id`s in react-native repo, but none of them are merged and they don't look like they will be merged soon :( example: https://github.com/facebook/react-native/pull/18527 
https://github.com/facebook/react-native/pull/9942

So this is not the ideal situation I should say. Appium forces us to use `accessibilityLabel`s, but  this value is visible to users, it is forcing us to enable accessibility on some elements we'd normally wouldn't need/want to. And also we can't use non-localized values for `accessibilityLabel`s since this value is visible to users.

To sum up, the need to use `accessibilityLabel`s forces us to use localized words on UI tests also.

Reply:
> I saw several PRs that are proposing to change this behavior and translating testIDs into resource-ids in react-native repo, but none of them are merged and they don't look like they will be merged soon :( 

Right! I came across a couple of those, we can watch and see if they get merged and then make an update after. 

> In this case we should continue using localized values, but, we should be using localized values while writing UI tests also.

Yeah! we should do this to get that covered once we starting testing across locales. I'll push an update to localize the query strings